gtk/src/default/gtk-3.20/_common.scss

```diff
@@ -1599,7 +1606,7 @@ headerbar {
   button.toggle:checked {
 
     background: if($variant == 'light', image(darken($bg_color, 17%)), image(darken($bg_color, 9%)));
-    border-color: darken($borders_color, 6%);
+    border-color: darken($borders_color, 6%); // Yaru change: with our paper buttons we dont need this
```

Comment:
what do you mean with "paper buttons"?

Reply:
I mean this slightly liftet buttons we carry with us since yaru 1.0 :) 
Adwaita has a gradient, which makes it look a bit bulky, but less shadow at the bottom.
![grafik](https://user-images.githubusercontent.com/15329494/93750145-5040ab80-fbfb-11ea-8bca-0253408a13c2.png)

Reply:
Flat buttons? Gradient-less buttons?